0033-relationship-between-protocols/0033-relationship-between-protocols.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Relationship between Protocols
+draft: 1
+---
+
+# Relationship between Protocols
+
+## Prerequisites
+This document assumes the reader is familiar with the following document:
+
+- [Interledger Architecture](../0001-interledger-architecture/0001-interledger-architecture.md)
+
+## Terminology
+- A **node** is a participant in an Interledger network. It may be a [connector](../0001-interledger-architecture/0001-interledger-architecture.md#connectors), a sender or a receiver.
+- A **payment** is sending value from one to another but it doesn't mean settled necessarily that is explained below. In the Interledger network, a payment could mean a **node-to-node** payment or an **end-to-end** payment.
+  - A **node-to-node** payment means the intention of transferring value of a node (a connector, an endpoint sender, or an endpoint receiver) to another node which is directly connected to the sender node.
+  - An **end-to-end** payment means the intention of transferring value of an endpoint sender to an endpoint receiver.
+- A **settlement** is ensuring that one has the paid value. For instance, executing a transaction on a certain ledger that the payee has an account on.
+  - A **node-to-node** payment is settled by means that the two nodes agreed upon. It is usually a ledger that tracks the amount that both nodes have.
+  - An **end-to-end** payment is settled through a chain of settlements of intermediaries above.
+  - Using [STREAM](../0029-stream/0029-stream.md) protocol, payments on the Interledger network could be bidirectional, thus settlements could be as well.
```

Comment:
I don't think it's because of STREAM that payments can be sent in either direction. Arguably, normal ILP enables that

Reply:
I'm sorry, that was misleading. Amended.